components/news_summary_process.py

```
@@ -0,0 +1,120 @@
+            st.info(f"기사 자동 저장에 실패하였습니다 ({e})")
+        if summary == "error1":
+            st.error("언어 감지에 실패하였습니다 (지원 언어: 영어, 한국어)")
+        elif summary == "error2":
+            st.error("지원하는 언어가 아닙니다 (지원 언어: 영어, 한국어)")
+        else:
+            st.success("✅ 생성 완료 !")
+            st.write(f"**{title}**")
+            st.write(summary)
+            st.write(f"공백 포함 요약문 길이: `{len(summary)}` | 분야: `{topic}`")
+
+            # 기존에 요약을 진행한 기사 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)